### ISSUE TITLE
namespacing a class requires that id's be prepended with the module

### ISSUE THREAD

**Issue by pedelman** [balvig/spyke#37]
I have another question related to associations. This is related to an example like this:

```rb
module Silly
  class State < Spyke::Base
    include_root_in_json false
    uri 'countries/:silly_country_id/states/(:id)'
    belongs_to :country
  end
end

module Silly
  class Country < Spyke::Base
    include_root_in_json false
    uri 'countries/(:id)'
    has_many :states, class_name: 'Silly::State', uri: 'countries/:silly_country_id/states/(:id)'
  end
end
```

```rb
Silly::State.where(:silly_country_id => 226).find(1).country
Silly::Country.find(226).states.find(1).country
```

Is there a way to have it so these ```:silly_country_id``` can just be ```:country_id```? The namespacing seems to make it so that it makes an assumption about the structure of the keys.

**Comment by pedelman** [comment]
Also found you have a similar line in your test fixtures:

```rb
module Cookbook
  class Tip < Spyke::Base
    uri 'tips/(:id)'
    has_many :likes, class_name: 'Cookbook::Like', uri: 'tips/:cookbook_tip_id/likes/(:id)'
    ....
  end
end
```

Is it possible to make the :cookbook_tip_id into just a tip_id?

**Comment by balvig** [comment]
Fixed in 2.0.1. Thanks for reporting :smile:

**Comment by jontebol** [comment]
This is exactly what I meant by my comment in 7e780c58271cdbbb3f472e147a51c97a0cfbfb50. Glad to see it fixed!
